modules/utxo-lib/test/integration_local_rpc/generate/fixtures.ts

```diff
@@ -16,7 +16,9 @@ export type Protocol = {
 export function getProtocolVersions(network: Network): number[] {
   switch (getMainnet(network)) {
     case networks.zcash:
-      return [400, 450, 500];
+      // FIXME: re-enable protocol version 500
```

Comment:
What's the impact if this doesn't get fixed by the next stable release?

Reply:
from what I can tell: lower test coverage for transactions types we probably don't need yet